src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -63,11 +63,10 @@ public void configureIndexWithInvalidIndexName() {
     @Test
     public void configureIndexExceedingQuota() {
         try {
-            controlPlaneClient.configureIndex(indexName, 400);
-
+            controlPlaneClient.configureIndex(indexName, 600);
```

Comment:
I think this was due to @haruska bumping our project quota, at least it seems like that's the case.